### PR TITLE
ERS-865 fishing report dto to flux

### PR DESCRIPTION
**Pull request by RobinMilding**
This pull request is about receiving a fishing report in EFR format, converting it to FLUX format, storing it in the db and putting an "ACK" message on the outgoing queue.

### CODE REVIEW

**Comment by RobinMilding** [comment]
test me

**Comment by RobinMilding** [comment]
test me

**Comment by RobinMilding** [comment]
test me god dammit

**Comment by RobinMilding** [comment]
test me

**Comment by RobinMilding** [comment]
test me once again you beautiful machine

**Comment by RobinMilding** [comment]
test me again, I promise it works now!

File: application/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/dto/efrbackend/AbstractModelObject.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
+package eu.europa.ec.fisheries.uvms.activity.service.dto.efrbackend;
+
+import javax.validation.constraints.NotNull;
+import java.time.Instant;
+
+/**
+ * Copied from the EFR Backend project, these DTO classes are how it sends the
+ * reports of completed fishing trips to this module
+ */
+public abstract class AbstractModelObject {
```
Comment (psjobergprivat) [inline application/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/dto/efrbackend/AbstractModelObject.java:10]: It's great that all these will go away!

**Review by psjobergprivat — APPROVED** [review]

**Pull request merged**